README.md

```diff
@@ -11,67 +11,32 @@ This crate makes it easy to develop Rust software that leverages **safe**,
-- `uefi`: High-level wrapper around various low-level UEFI APIs. \
-  Offers various optional features for typical Rust convenience, such as a
-  Logger and an Allocator. (_This is what you are usually looking for!_)
-- `uefi-macros`: Helper macros. Used by `uefi`.
-
-
-You can use the abstractions for example to:
-
-- create OS-specific loaders and leverage UEFI boot service
-- access UEFI runtime services from an OS
-
-[UEFI]: https://en.wikipedia.org/wiki/Unified_Extensible_Firmware_Interface
-
 ![UEFI App running in QEMU](https://imgur.com/SFPSVuO.png)
 Screenshot of an application running in QEMU on an UEFI firmware that leverages
 our Rust library.
 
-## User Documentation
+## API and User Documentation
+<!-- This section is duplicated with uefi/README.md -->
 
-<!-- KEEP IN SYNC WITH uefi/README -->
-
-For a quick start, please check out [the UEFI application template](template).
-
-The [uefi-rs book] contains a tutorial, how-tos, and overviews of some important
-UEFI concepts. Reference documentation for the various crates can be found on
-[docs.rs]:
-
-- [docs.rs/uefi](https://docs.rs/uefi)
-- [docs.rs/uefi-macros](https://docs.rs/uefi-macros)
-- [docs.rs/uefi-raw](https://docs.rs/uefi-raw)
-
-For additional information, refer to the [UEFI specification][spec].
-
-[spec]: https://uefi.org/specs/UEFI/2.10
-[uefi-rs book]: https://rust-osdev.github.io/uefi-rs/HEAD
-[docs.rs]: https://docs.rs
-
-### MSRV
-
-See the [uefi package's README](uefi/README.md#MSRV).
+The main contribution of this project is the [`uefi` crate](./uefi/README.md).
+Please refer to [docs.rs](https://docs.rs/uefi) for comprehensive documentation
+of the **latest stable release**. The latest not necessarily yet published
+documentation can be found in [`src/lib.rs`](./uefi/src/lib.rs), which can also
+be locally viewed by running `$ cargo xtask doc --open`.
 
 ## Developer Guide
 
-### Project structure
-
-This project contains multiple sub-crates:
+### Repository Structure
 
-- `uefi`: defines the standard UEFI tables / interfaces.
-  The objective is to stay unopinionated and safely wrap most interfaces.
-  Additional opinionated features (such as a Logger) are feature-gated.
+This repository provides various crates:
 
-- `uefi-macros`: procedural macros that are used to derive some traits
-  in `uefi`.
+- [`uefi-raw`](/uefi-raw/README.md): Raw Rust UEFI bindings for basic structures and functions.
+- [`uefi`](/uefi/README.md): High-level wrapper around various low-level UEFI APIs. \
+  Offers various optional features for typical Rust convenience, such as a
+  Logger and an Allocator.
+  This is the **main contribution** of this project.
+- [`uefi-macros`](/uefi-macros/README.md): Helper macros used by `uefi`.
+- [`uefi-test-runner`](/uefi-test-runner/README.md): a UEFI application that runs our unit / integration tests.
 
 - `uefi-raw`: raw types that closely match the definitions in the UEFI
```

Comment:
Duplicate entry, `uefi-raw` is now linked/described at the top of the list.

Reply:
Sorry, an artifact of all the major rebases this PR experienced over the last weeks. Fixed!